www/assets/css/theme.css

```diff
@@ -378,3 +378,99 @@ code { /* character, digrams, trigrams */
     color: var(--fg-banner);
   }
 }
+
+
+/**
+ * ╭───────────────────────────────────────────────────────────────╮
+ * │                          Comments                             │
+ * ╰───────────────────────────────────────────────────────────────╯
+**/
+
+section#comments #comments-wrapper {
```

Comment:
nitpicks, mostly for consistency : 
- `#comments-wrapper` is enough here
- empty lines between CSS rules please (much easier to navigate with Vim)
- space after the colon sign please (`: ` rather than `:`)